### ISSUE TITLE
Disable Spamassassin Subject Tag

### ISSUE THREAD

**Issue by elbracht** [tomav/docker-mailserver#589]
Is there a way to disable the `sa_spam_subject_tag` in `/etc/amavis/conf.d/20-debian_defaults`?

Spamassassin shall not add \*\*\*SPAM*** to the mail subject.

**Comment by tomav** [comment]
Hi @elbracht, true [this file](https://github.com/tomav/docker-mailserver/wiki/FAQ-and-Tips#how-can-i-manage-my-custom-spamassassin-rules) and let me know (not tested)

**Comment by elbracht** [comment]
I found the following on https://wiki.apache.org/spamassassin/SubjectRewrite:

```
Warning: if you are running SA through amavisd or qmail-scanner, those apps do their own message rewriting and SpamAssassin cannot perform these rewrites.
```

Is it possible to change the amavis configuration outside the container?

**Comment by elbracht** [comment]
Implemented in #596
